scripts/install/proctoring.rdf

```diff
@@ -44,4 +46,26 @@
     <rdfs:comment xml:lang="en-US"><![CDATA[Canceled status of a delivery execution]]></rdfs:comment>
     <rdfs:subClassOf rdf:resource="http://www.tao.lu/Ontologies/TAODelivery.rdf#DeliveryExecutionStatus"/>
   </rdf:Description>
+
+    <rdf:Description rdf:about="http://www.tao.lu/Ontologies/TAODelivery.rdf#ProctorAccessible">
+      <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
+      <rdfs:label xml:lang="en-US"><![CDATA[Accessible by Proctor]]></rdfs:label>
+      <rdfs:comment xml:lang="en-US"><![CDATA[Accessible by Proctor]]></rdfs:comment>
+      <widget:widget rdf:resource="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#CheckBox"/>
+      <generis:is_language_dependent rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#False"/>
+      <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/TAODelivery.rdf#Delivery"/>
+      <rdfs:range rdf:resource="http://www.tao.lu/Ontologies/TAODelivery.rdf#CheckMode"/>
```

Comment:
Range could be `<rdfs:range rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#Boolean"/>` and  `http://www.tao.lu/Ontologies/generis.rdf#True`/`http://www.tao.lu/Ontologies/generis.rdf#False` may be used to enable/disable proctoring.